### ISSUE TITLE
Fix "tabbing" through LenaSYS

### ISSUE THREAD

**Issue by a20vicad** [HGustavs/LenaSYS#12336]
Related to #11794
Some pages don't have a logical tab order, and some buttons are not accessible through "tabs".
![image](https://user-images.githubusercontent.com/102599327/167578274-e406e139-ac54-458b-b379-3adcd3fe8711.png)

**Comment by HGustavs** [comment]
Yes, this is an accessibility feature, and is a worthwhile cause if we have people without work tasks.

Also this is true for all dialogs in lenasys, is the tab ordering logical?
